src/main/java/org/heigit/ohsome/ohsomeapi/inputprocessing/InputProcessor.java

```diff
@@ -100,10 +104,6 @@ public InputProcessor(HttpServletRequest servletRequest, boolean isSnapshot, boo
     this.requestParameters = servletRequest.getParameterMap();
   }
 
-  public InputProcessor(ProcessingData processingData) {
```

Comment:
If we want to use Lombok, I think we should establish in which cases Lombok has to be used and in which cases avoided.
This is one of more cases in which the use of Lombok is possible but not necessary convenient.

Reply:
In my opinion, the other usage examples here in this branch are totally fitting, e.g. hiding the standard getter/setter writing, as well as the "standard constructor usage" meaning cases where you would use ```@AllArgsConstructor```.

But in cases like here, when we have more than one constructor and it also includes some processing, I'd rather avoid it.
Curious what @rtroilo or @tyrasd would have to say about that too.

Reply:
There are another cases which could be a little bit misleading, for example [here](https://github.com/GIScience/ohsome-api/blob/64b3d53e58cf4d368625811528fa115f5b703b2f/src/main/java/org/heigit/ohsome/ohsomeapi/executor/ExecutionUtils.java#L94)
It is possible to use only private and not-initialized variables with the annotation @RequiredArgsConstructor. Since the constructor here takes only `ProcessingData processingData;`, which is also the only variable of this kind in the class, this annotation works for the purpose but it could mislead the developer.